python/tank/platform/software_launcher.py

```diff
@@ -32,16 +34,34 @@
 core_logger = LogManager.get_logger(__name__)
 
 
-def create_engine_launcher(tk, context, engine_name):
+def _format(template, tokens):
```

Comment:
The unit tests on Python 2.5 will fail if we don't do this, so we'll have to keep it for now. I'll move it inside the class however.